src/electionguard/guardian.py

```diff
@@ -169,12 +176,25 @@ def __init__(
         self.id = id
         self.sequence_order = sequence_order
         self.set_ceremony_details(number_of_guardians, quorum)
-        self._backups_to_share = {}
-        self._guardian_election_public_keys = {}
-        self._guardian_election_partial_key_backups = {}
-        self._guardian_election_partial_key_verifications = {}
+        self._backups_to_share = {} if backups_to_share is None else backups_to_share
```

Comment:
I tried setting the default value of these new params to `{}` because it massively reduced cyclomatic complexity, but it produced linting errors, thus all these ternaries. I'd love to hear if there's a simpler way to do this.